source/core/index.ts

```diff
@@ -861,10 +861,12 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 								if (event === 'error') {
 									result.catch(handler);
 								} else if (event === 'abort') {
+									// The empty catch is needed here in case when
+									// it rejects before it's `await`ed in `_makeRequest`.
 									// eslint-disable-next-line promise/prefer-await-to-then
-									void result.then((request: unknown): void => {
+									result.then((request: unknown): void => {
 										(request as ClientRequest).once('abort', handler);
-									});
+									}).catch(() => {});
```

Comment:
```js
(async () => {
	try {
		let request = await result;
		(request as ClientRequest).once('abort', handler);
	} catch {}
})();
```

Reply:
I haven't tried, but if this works, use this:

```js
(async () => {
	try {
		let request: ClientRequest = await result;
		request.once('abort', handler);
	} catch {}
})();
```

Reply:
Should definitely work. Will fix.